### ISSUE TITLE
Please add api call to get all tickers

### ISSUE THREAD

**Issue by venil7** [xemwebe/yahoo_finance_api#8]
At the moment tickers dont come from API,
need to be added manually

is it possible to fetch tickers from API ?

PS: after little research i learned that you can make the following request
```
https://uk.finance.yahoo.com/_finance_doubledown/api/resource/searchassist;searchTerm={PARTIAL_SYMBOL_HERE}
```

**Comment by xemwebe** [comment]
Thanks for the suggestion, that shouldn't be to complicated and would indeed be helpful. 
I will have closer look at this.

**Comment by xemwebe** [comment]
I  have just pushed a new branch [add_ticker_search](https://github.com/xemwebe/yahoo_finance_api/tree/add_ticker_search), which should do just what you requested. I have included as much information from the request response as seems useful, please let me know if something is missing before I merge it to master.

**Comment by venil7** [comment]
@xemwebe That looks good to me, thanks for your effort!!

**Comment by xemwebe** [comment]
I have merged the branch into master and published the crate with version number 1.1.0, therefore I close this issue.
